src/hamster/edit_activity.py

```diff
@@ -329,30 +330,31 @@ def validate_fields(self):
         """Check fields information.
 
         Update gui status about entry and description validity.
-        Try to merge date, activity and description informations.
+        Try to merge day, activity and description informations.
 
         Return the consolidated fact if successful, or None.
         """
         fact = self.fact
 
         now = dt.datetime.now()
-        self.get_widget("button-next-day").set_sensitive(self.date < now.date())
+        today = now.hday()
```

Comment:
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from datetime import datetime
>>> datetime.now
<built-in method now of type object at 0xa346e0>
>>> datetime.now()
datetime.datetime(2020, 4, 27, 13, 16, 56, 954984)
>>> datetime.now().hday()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'datetime.datetime' object has no attribute 'hday'
```

Reply:
What this theoretical, or based on an actual test ?
Please look at the imports: `from hamster.lib import datetime as dt`

```bash
~/share/prog/python/hamster/src>GSETTINGS_SCHEMA_DIR=../build/data   python3 
Python 3.6.10 (default, Jan 16 2020, 09:12:04) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from hamster.lib import datetime as dt
>>> now = dt.datetime.now()
>>> now.hday()
hday(2020, 4, 27)
```

Reply:
ah okay, that's where the object came from! I didn't recognize `hday`, was just trying to be helpful, thinking this was part of the untested code :)

Reply:
> Maybe some renaming could help here at some point.

And forget about compatibility with existing external code ?